src/app/qtsingleapplication/qtlocalpeer.cpp

```diff
@@ -191,6 +192,12 @@ void QtLocalPeer::receiveConnection()
     QByteArray uMsg;
     quint32 remaining;
     ds >> remaining;
+    if (remaining > 65535) {
```

Comment:
IIRC, some malware can connect to this socket and send some big data here? If not, then what is it? Please describe.

Reply:
>IIRC, some malware can connect to this socket and send some big data here?

This is correct, theoretically it can hang qbt or even the system with big allocations.